engine-cdi/core/pom.xml

```diff
@@ -224,14 +155,6 @@
             <dependency>org.camunda.bpm:camunda-bpm-archunit</dependency>
           </dependenciesToScan>
         </configuration>
-        <dependencies>
-          <!-- Force using the JUnit provider -->
-          <dependency>
-            <groupId>org.apache.maven.surefire</groupId>
-            <artifactId>surefire-junit47</artifactId>
-            <version>2.22.2</version>
-          </dependency>
-        </dependencies>
```

Comment:
🔧 We don't need the `create-test-jar-weld` execution anymore. So `maven-jar-plugin` definition can be removed below, right? Or we can clean up the weld3 comment.

Reply:
Good point